docs/cloud/features/test-replay.mdx

```diff
@@ -114,6 +132,111 @@ Toggle each test attempt, play back errors at a variety of speeds or scrub throu
   alt="Test Replay scrubber controls"
 />
 
+## How Test Replay Works
+
+Our aim is to create an impactful debugging experience that covers the most ground for users.
+Cypress captures a wide range of data during end-to-end and component test runs.
+This data is then uploaded and stored in Cypress Cloud and replayed for debugging through Test Replay.
+
+### What is captured
+
+- Test commands
+- Network requests
+- Console logs
+- JavaScript errors
+- Element rendering
+- DOM mutations
+- CSS styles
+- SVG elements
+- Iframes
+- Shadow DOM
+- Canvas elements (v13.5.0+, please contact us at support@cypress.io to enable capturing canvas elements)
+
+### What is not captured
+
+Currently Test Replay does not support the following:
+
+- Tests recorded in Cypress App below v13.
+- WebKit browser (Safari) ([see issue](https://github.com/cypress-io/cypress/issues/28895))
+- Firefox browser ([see issue](https://github.com/cypress-io/cypress/issues/28894))
+- Video ([see issue](https://github.com/cypress-io/cypress/issues/29903))
+- Audio ([see issue](https://github.com/cypress-io/cypress/issues/29905))
+- Objects that do not have `type="image/svg+xml"` ([see issue](https://github.com/cypress-io/cypress/issues/29904))
+- Shadow DOM with `slotAssignment="manual"` ([see issue](https://github.com/cypress-io/cypress/issues/29107))
+- Shadow DOM canvas ([see issue](https://github.com/cypress-io/cypress/issues/30438))
+- Server sent events ([see issue](https://github.com/cypress-io/cypress/issues/29908))
+- Web sockets ([see issue](https://github.com/cypress-io/cypress/issues/29907))
+- localStorage & sessionStorage ([see issue](https://github.com/cypress-io/cypress/issues/29909))
+- Cookies ([see issue](https://github.com/cypress-io/cypress/issues/29910))
+- Console props of Cypress commands outside of `cy.request()` ([see issue](https://github.com/cypress-io/cypress/issues/29911))
```

Comment:
Do we want to call out that we do not capture network traffic of `cy.request()` either?

Reply:
Addressed here [`7ec343a` (#5985)](https://github.com/cypress-io/cypress-documentation/pull/5985/commits/7ec343a678e0c8096d598249fe0f73823421f37a) I created a public issue to track this also.